contrib/zfs-tests-color.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```

Comment:
One last thing.  Let's put this under `scripts/` with the other assorted support scripts instead of `contrib/`.  I just noticed you'll also need to add it to the `Makefile.am`.